src/devices/Dhtxx/DhtType.cs

```diff
@@ -9,7 +9,15 @@ namespace Iot.Device.DHTxx
     /// </summary>
     public enum DhtType
     {
-        Dht11,
-        Dht22
+        DHT11,
```

Comment:
`Dht11`, `Dht12`...

Reply:
The name of this kind of sensor is DHTxx. I don't think it's necessary to use Dhtxx.😄

Reply:
I personally prefer to use exact casing as you did above but our [coding guidelines](https://github.com/dotnet/iot/blob/master/Documentation/Coding-guidelines.md) is fairly clear about using `Dht` :( Once we have enough evidence this doesn't work for this repo we can reiterate :smile: